.env

```diff
@@ -0,0 +1,2 @@
+# pipenv configs
+PIPENV_VENV_IN_PROJECT=1
```

Comment:
I don't really have a preference myself, but I'm curious, is there any reason to overwrite the default?